scripts/fns.js

```
@@ -0,0 +1,119 @@
+   * }[]}
+   */
+  const asciiTable = [];
+  
+  for (const item of allItems) {
+    const itemPath = path.join(srcDir, item);
+    const itemZipPath = path.join(isMV3 ? 'dist/' : 'dist-mv2/', item);
+
+    const stat = fs.statSync(itemPath);
+
+    if (stat.isDirectory()) {
+      await addDirectoryToZip(itemPath, itemZipPath);
```

ESLint

[error] 'addDirectoryToZip' is not defined.

(no-undef)